python/cudf/cudf/core/frame.py

```diff
@@ -1341,10 +1341,7 @@ def _apply(self, func):
         """
         Apply `func` across the rows of the frame.
         """
-        output_dtype, ptx = cudf.core.udf.pipeline.compile_masked_udf(
-            func, self.dtypes
-        )
-        result = cudf._lib.transform.masked_udf(self, ptx, output_dtype)
+        result = cudf.core.udf.pipeline.udf_pipeline(self, func)
```

Comment:
I think you need:

```
import cudf.core.udf.pipeline
```

just before this (I did).

Reply:
Should be fixed